udata_front/theme/gouvfr/datagouv-components/src/components/ReuseCard/ReuseCard.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <article class="fr-card fr-card--no-icon fr-enlarge-link">
+    <div class="fr-card__body">
+      <div class="fr-card__content fr-px-2w fr-py-1v">
+        <h3 class="fr-card__title fr-text--bold fr-mt-1v fr-mb-0">
+          <AppLink :to="reuseUrl">
+            {{ truncate(reuse.title, 55) }}
+          </AppLink>
+        </h3>
+        <div class="fr-card__desc fr-mt-1v text-mention-grey">
+          <p class="fr-mb-0">
+            <template v-if="reuse.organization">
+              <span class="not-enlarged dash-after-sm">
+                <AppLink class="fr-link" v-if="organizationUrl" :to="organizationUrl">
+                  <OrganizationNameWithCertificate :organization="reuse.organization" />
+                </AppLink>
+                <OrganizationNameWithCertificate v-else :organization="reuse.organization" />
+              </span>
+            </template>
+            <span class="dash-after-sm" v-else>
+              {{ ownerName }}
+            </span>
+            {{ t('Published {date}', {date: formatRelativeIfRecentDate(reuse.created_at)}) }}
+          </p>
+          <div class="fr-grid-row fr-grid-row--middle fr-mt-1v fr-mr-1v">
+            <span class="dash-after-sm">
+              {{ reuseType }}
+            </span>
+            <p class="fr-text--sm fr-my-0">
+              <span class="fr-icon-star-line fr-icon--sm fr-px-1v" aria-hidden="true"></span>{{ reuse.metrics.followers }}
+            </p>
+            <p class="fr-text--sm fr-my-0">
+              <span class="fr-icon-eye-line fr-icon--sm fr-px-1v" aria-hidden="true"></span>{{ reuse.metrics.views }}
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="fr-card__header">
+      <div class="absolute top-0 fr-mt-2w fr-ml-2w">
```

Comment:
Can you add a test to hide this div if the reuse is neither private nor archived ?